test/test_helper.rb

```diff
@@ -12,7 +12,9 @@ class ActiveSupport::TestCase
   def before_setup
     # FIXME: make sure all users are deleted, sometimes records stick around
     # when tests are aborted
-    User.with_deleted.map(&:really_destroy!)
+    [:user, :volunteer, :client, :contact, :profile, :journal].each do |model|
+      model.to_s.classify.constantize.with_deleted.map(&:really_destroy!)
+    end
```

Comment:
You can also just do `[User, Volunteer, ...]` so you don't have to do `.to_s.classify.constantize` :)